scripts/purge-deployments.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euox pipefail
+
+export VALID_NAMESPACES=("hocs-gamma" "hocs-delta" "hocs-qax")
+
+current=$(kubectl config current-context)
+if [[ ${VALID_NAMESPACES[*]} =~ $current ]] ; then
+  echo "Current context is $current"
+else
+  echo "Unauthorised Namespace" >&2
+  exit 1
+fi
+
+echo 'Warning: this action will completely remove all the kubernetes deployments of '${current^^}'. It cannot be undone!'
+
+read -p 'Are you sure you want to remove all the kubernetes deployments (y/n)? ' yes_no
+
+if [[ $yes_no = 'y' || $yes_no = 'Y' ]]
+then
```

Comment:
You could do `kubectl delete --all deployments`. If you chain them (`kubectl delete deployment hocs-{info-service,casework,workflow,audit,...,case-creator}` it'll run a bit quicker.

` --wait=true` might also be interesting.

Do we need to tear down the other resources (e.g. services, network policies) too?

Reply:
Would `--all` not delete the toolbox too? Chaining does sound like a good solution.

Reply:
You could also do it based on label; e.g.  `kubectl delete deployments -l name=hocs-backend`. `kd` also lets you delete things.